product_template_multi_link/models/product_template_link.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2017-Today GRAP (http://www.grap.coop).
+# @author Sylvain LE GAL <https://twitter.com/legalsylvain>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from contextlib import contextmanager
+
+from psycopg2.extensions import AsIs
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductTemplateLink(models.Model):
+    _name = "product.template.link"
+    _order = "product_tmpl_id_left, product_tmpl_id_right"
+    _description = "Product link"
+
+    product_tmpl_id_left = fields.Many2one(
```

Comment:
what about `left_product_templ_id` and `right_product_tmpl_id` to respect convention?